doc/tutorials/content/sources/qt_visualizer/pclviewer.h

```diff
@@ -58,5 +68,10 @@ public Q_SLOTS:
 
 private:
   Ui::PCLViewer *ui;
-
+  /** @Brief QVTKWidget pointer */
+  #if VTK_MAJOR_VERSION > 8
```

Comment:
Use the PCL visualization header instead

Reply:
This would require to move the qvtk_compatability.h to visualization folder. Since the tutorials doesn't depend on apps folder (which is not part of the libraries).
Ie. make visualization have Qt as optional dependency.

Reply:
I'm not sure how to ensure if visualization was build with QT enabled in downsource projects? We would need to add a PCL_QVTK_FOUND variable to our PCLConfig.cmake?

Reply:
Ah, we could always include qvtk_compatability.h and in there, do a 
```
#if HAVE_QVTK 
  include stuff 
#else 
  #Error PCL not compiled with QVTK support 
#endif
```